builder/dockerfile/metrics.go

```diff
@@ -0,0 +1,31 @@
+package dockerfile
+
+import "github.com/docker/go-metrics"
+
+var (
+	buildsTriggered        metrics.Counter
+	buildsTriggeredFailed  metrics.LabeledCounter
+	buildsTriggeredSuccess metrics.Counter
+)
+
+func init() {
+	buildMetrics := metrics.NewNamespace("builder", "dockerfile", nil)
```

Comment:
never mind, I got it now: `buildMetrics := metrics.NewNamespace("builder", "", nil)`. I'll remove the `dockerfile` there.